packages/connector-kit/src/types.ts

```diff
@@ -58,17 +77,48 @@ export enum MessageTypes {
 
 export const messageTypesGuard = z.nativeEnum(MessageTypes);
 
-export type ConnectorMetadata = {
-  id: string;
-  target: string;
-  platform: Nullable<ConnectorPlatform>;
-  name: I18nPhrases;
-  logo: string;
-  logoDark: Nullable<string>;
-  description: I18nPhrases;
-  readme: string;
-  configTemplate: string;
-};
+export const fullConnectorMetadataGuard = z.object({
```

Comment:
no need to export?

Reply:
I think we need to merge metadata from code and DB, then use fullConnectorMetadataGuard.parse() to make sure the merged metadata is complete.

Reply:
Fixed this unnecessary export and removed redundant guard definition.